eng/pipelines/templates/steps/archetype-sdk-docs.yml

```diff
@@ -29,4 +29,13 @@ steps:
           -BinDirectory $(Build.BinariesDirectory)
           -DocGenDir ${{parameters.DocGenerationDir}}
           -ArtifactStagingDirectory '$(Build.ArtifactStagingDirectory)'
-          -verbose
\ No newline at end of file
+          -verbose
+    - task: PowerShell@2
```

Comment:
Instead of doing this as separate step in the yml file why not put it in the Generate-Api-Docs.ps1 directly?